server/test/unit/server/webservices/views/test_plugins.py

```diff
@@ -10,8 +10,56 @@
 
 
 class TestDistributorResourceView(unittest.TestCase):
-    pass
+    """
+    Tests for views for a single distributor.
+    """
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.plugins.generate_json_response')
+    @mock.patch('pulp.server.webservices.views.plugins.factory')
+    def test_get_distributor_resource_with_existing_distributor(self, mock_factory, mock_serial):
+        """
+        Distributor Resource should generate a json response from the distributor data.
+        """
+        mock_manager = mock.MagicMock()
+        mock_manager.distributors.return_value = [{'id': 'mock_distributor_1'},
+                                                  {'id': 'mock_distributor_2'}]
+        mock_factory.plugin_manager.return_value = mock_manager
+        request = mock.MagicMock()
+        request.get_full_path.return_value = '/mock/path/'
+
+        distributor_resource = DistributorResourceView()
+        response = distributor_resource.get(request, 'mock_distributor_2')
+
+        expected_content = {'id': 'mock_distributor_2', '_href': '/mock/path/'}
+        mock_serial.assert_called_once_with(expected_content)
+        self.assertTrue(response, mock_serial.return_value)
```

Comment:
hah, close, but not quite. What you have will just make sure that ``bool(response) == True``.

Fix: ``s/,/ is``